control/preimage/src/treasury_commands.rs

```diff
@@ -0,0 +1,221 @@
+use crate::{constants::*, TreasuryProposal2024Args};
+
+use crate::helpers::utility_force_batch;
+use crate::relay_runtime::api::runtime_types::{
+    pallet_treasury,
+    polkadot_runtime_common::impls::VersionedLocatableAsset,
+    staging_xcm::v4::{
+        asset::AssetId, junction::Junction, junctions::Junctions, location::Location,
+    },
+    xcm::VersionedLocation,
+};
+use crate::relay_runtime::RuntimeCall as RelayRuntimeCall;
+use polkadot_runtime_constants::currency::UNITS;
+use polkadot_runtime_constants::time::DAYS;
```

Comment:
this uses block numbers under the hood right? the polkadot library already has a day hardcoded in number of blocks?

Reply:
yep, DAYS is a function of block numbers.